docs-source/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -1,11 +1,11 @@
----
+PersistentVolumeClaim---
```

Comment:
Just curious... What does this mean at the top of the file?

Reply:
I'd say "that was a boo-boo", an errant replacement that needs to be fixed.